bot/cogs/polling.py

```diff
@@ -34,32 +34,33 @@ async def on_ready(self):
 
     async def new_poll(self, pollings, server: str, poll: str):
         while True:
-            #try:
+            # try:
             if pollings.get("type") == "player":
                 interval = float(pollings.get("polling_interval")) * 60
                 await asyncio.sleep(interval)
                 logging.info(f"Executing Task {poll} on server {server}")
                 try:
                     state, ctx = await self.player_polling(ctx, pollings, server, state)
```

Comment:
Order of arguments seems wrong here. `ctx` is at the back now :)

Reply:
I'm confused. What do you mean?